.github/workflows/autobuild.yaml

```diff
@@ -126,11 +210,19 @@ jobs:
         ninja -C grpc-build install
     - name: Fetch target commit
       uses: actions/download-artifact@v1
-      with: { name: release }
+      with:
+        name:
+          env
+    - name: Fetch release info
+      uses: actions/download-artifact@v1
+      with:
+        name:
+          release
     - name: Set target commit
       run: |
-        cat release/commit.env >> $GITHUB_ENV
-        echo "CLANGD_DIR=clangd_${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+        cat env/commit.env >> $GITHUB_ENV
+        cat release/release.env >> $GITHUB_ENV
+        echo "CLANGD_DIR=clangd_${{ env.TAG_NAME }}" >> $GITHUB_ENV
```

Comment:
nit: you can also set this via https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenv.

Reply:
But this wouldn't work with the non-plain text :(